apps/react-workshop/src/Alert.test.ts

```diff
@@ -18,6 +18,8 @@ describe('Alert', () => {
     it(testName, () => {
       const id = Cypress.storyId(storyPath, testName);
       cy.visit('/', { qs: { mode: 'preview', story: id } });
+      cy.wait(500);
```

Comment:
Can you add a `TODO` comment next to all of these waits? This is slowing down our tests and we need to investigate it later.